### DISCUSSION TITLE
Can you make a separate build with undervolting functions only?

### DISCUSSION THREAD

**Discussion by chockokruasan** [seerge/g-helper#3238] [Ideas]
Hello @seerge can you make a build with undervolting cpu/igpu functions ONLY (without other controls and options) for users who stay with AC, but want to undervol cpu? Is it possible for donation maybe? Thanks!

**Comment by seerge** [comment]
@chockokruasan hello, this "version" already exists and can do much more. It's called [UXTU](https://github.com/JamesCJ60/Universal-x86-Tuning-Utility)

**Reply by chockokruasan** [reply]
Yes, but this software works only with Virtualization off, but I need this tehnology on for work. Ghelper UV is easier also.

**Reply by seerge** [reply]
@chockokruasan then use G-Helper and don't apply any custom settings except UV. By default G-Helper won't set anything, except one of 3 predefined and stored in BIOS modes (that you use anyway).

**Reply by chockokruasan** [reply]
@seerge thanks, other options like keyboard lighting, GPU settings, options in "extra" tab, something else doesn not confict with AC or MyAsus?

**Reply by seerge** [reply]
@chockokruasan if you don't change them - they don't. But keep in mind that this is still extremely exotic usecase. You can turn off [Aura init command via config](https://github.com/seerge/g-helper/wiki/Power-user-settings#skip-keyboard-aura-initialisation-on-startup)
